_security-plugin/configuration/index.md

```diff
@@ -13,11 +13,11 @@ redirect_from:
 The plugin includes demo certificates so that you can get up and running quickly, but before using OpenSearch in a production environment, you must configure it manually:
```

Comment:
I recommend breaking this sentence into two sentences: "The plugin includes demo certificates so that you can get up and running quickly. Before using OpenSearch in a production environment, you must configure it manually:"